Samples/AudioPreprocessing/AudioPreprocessing/ViewModel/PreprocessViewModel.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using AudioPreprocessing.Model;
+using Microsoft.Toolkit.Mvvm;
+using Microsoft.Toolkit.Mvvm.Input;
+using MvvmDialogs;
+using MvvmDialogs.FrameworkDialogs.OpenFile;
+
+namespace AudioPreprocessing.ViewModel
+{
+    // Implements INotifyPropertyChanged interface to support bindings
+    public class PreprocessViewModel : INotifyPropertyChanged
+    {
+        private readonly IDialogService dialogService;
+        private string helloString;
+        private string melSpecPath;
+
+        public PreprocessViewModel()
+        {
+            PreprocessModel helloWorldModel = new PreprocessModel();
+            melSpecPath = helloWorldModel.MelSpecPath;
+            helloString = helloWorldModel.ImportantInfo;
+            OpenFileCommand = new RelayCommand(OpenFile);
+        }
+
+        public string MelSpecPath
+        {
+            get { return melSpecPath; }
+            set { melSpecPath = value; OnPropertyChanged(); }
+        }
+
+        public ICommand OpenFileCommand { get; }
+
+        private void OpenFile()
+        {
+         
+            var settings = new OpenFileDialogSettings
+            {
+                Title = "This Is The Title",
+                //InitialDirectory = IOPath.GetDirectoryName(Assembly.GetExecutingAssembly().Location),
+                Filter = "Text Documents (*.txt)|*.txt|All Files (*.*)|*.*"
+            };
+
+            bool? success = dialogService.ShowOpenFileDialog(this, settings);
+            if (success == true)
```

Comment:
someone can correct me if I'm out of date, but I believe we avoid the "== true" and just use, "if (success)" in cases like this

Reply:
I took this out completely! Will check in the final version of the file chooser soon!